docs/helm-chart-checks.md

```diff
@@ -3,9 +3,10 @@
 Helm chart checks are a set of checks against which the Red Hat Helm chart-verifier tool verifies and validates whether a Helm chart is qualified for a certification from Red Hat. These checks contain metadata that have certain parameters and values with which a Helm chart must comply to be certified by Red Hat. A Red Hat-certified Helm chart qualifies in terms of readiness for distribution in the [Red Hat OpenShift Helm Chart Repository](https://github.com/openshift-helm-charts).
 
 ## Key features
-- You can execute the checks individually. Individual checks can be included or excluded during the verification process using the command line options.
+- You can execute all of the mandatory checks.
+  - Individual checks can be included or excluded during the verification process using the command line options.
```

Comment:
Given your first sentence in this bullet list is first person, consider making the second sentence first person voice as well (instead of passive voice). This is simply for consistency. Not that there's anything wrong or even bad with how it's written now.
Ex:
You can execute the checks individually. Include or exclude individual checks during the verification process by using the command line options.

Reply:
Done. thanks :)